scripts/verify-webservice-image-digest.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+"""
+
+This script is meant to help with verifying that the image created in CircleCI and pushed to quay
+match what we download via the Docker client. This ensures there has been no "registry poisoning"
+or modifications to our Docker image since we created it.
+
+Depending on how you have configured your Docker client you may need to run this script via sudo,
+it will download and inspect the Docker image as required.
+
+"""
+
+import argparse
+import requests
+import subprocess
+
+
+parser = argparse.ArgumentParser(
+  description='Verify that CircleCI (via S3) and quay.io have the same image digest for the Dockstore Webservice')
+parser.add_argument('tag', type=str,
+                    help='The git tag (or branch) you would like to verify')
+
+args = parser.parse_args()
+
+DOCKER_TAG_BASE = "quay.io/dockstore/dockstore-webservice"
+
+def get_docker_digest(image_path):
+  # takes an image path and returns the digest as reported by Docker CLI
+  message = ""
+  print("Pulling the image {}...".format(image_path))
+  try:
+      subprocess.check_output(["docker", "pull", image_path])
+      message = str(subprocess.check_output(["docker", "inspect", "--format='{{index .RepoDigests 0}}'", image_path]))
+  except subprocess.CalledProcessError as err:
+      print(err)
+      exit(1)
+  loc = message.find('sha256')
+  return message[loc+7:loc+71]
+
+def get_commit_from_github(tagOrBranch):
```

Comment:
Very small (ignore-ably small) nit: camelCase amongst the snake case

Reply:
Thanks